tabpy-server/tabpy_server/common/endpoint_file_mgr.py

```diff
@@ -15,7 +15,7 @@
 
 logger = logging.getLogger(__name__)
 
-_name_checker = _compile(r'^[a-zA-Z0-9-_\ ]+$')
+_name_checker = _compile('^[a-zA-Z0-9-_\ ]+$')
```

Comment:
I think you just want to use the built in character class of `\w` for this

Reply:
Yes, we'll have another fix for regex. With `r` prefix models deployment doesn't work so need to look into it.